2022-03-13 18:51:45,780 INFO    MainThread:13351 [wandb_setup.py:_flush():75] Loading settings from /nethome/mrungta8/.config/wandb/settings
2022-03-13 18:51:45,781 INFO    MainThread:13351 [wandb_setup.py:_flush():75] Loading settings from /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning/wandb/settings
2022-03-13 18:51:45,781 INFO    MainThread:13351 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-13 18:51:45,781 INFO    MainThread:13351 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'Mukund/RE_improved_baseline/meta_learning/train_tacred.py', 'program': 'train_tacred.py'}
2022-03-13 18:51:45,781 INFO    MainThread:13351 [wandb_init.py:_log_setup():386] Logging user logs to /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning/wandb/run-20220313_185145-3n6zhst4/logs/debug.log
2022-03-13 18:51:45,781 INFO    MainThread:13351 [wandb_init.py:_log_setup():387] Logging internal logs to /nethome/mrungta8/project/ProjectPURE/Mukund/RE_improved_baseline/meta_learning/wandb/run-20220313_185145-3n6zhst4/logs/debug-internal.log
2022-03-13 18:51:45,781 INFO    MainThread:13351 [wandb_init.py:init():420] calling init triggers
2022-03-13 18:51:45,782 INFO    MainThread:13351 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-03-13 18:51:45,782 INFO    MainThread:13351 [wandb_init.py:init():471] starting backend
2022-03-13 18:51:45,782 INFO    MainThread:13351 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-13 18:51:45,796 INFO    MainThread:13351 [backend.py:ensure_launched():219] starting backend process...
2022-03-13 18:51:45,808 INFO    MainThread:13351 [backend.py:ensure_launched():225] started backend process with pid: 13493
2022-03-13 18:51:45,809 INFO    MainThread:13351 [wandb_init.py:init():480] backend started and connected
2022-03-13 18:51:45,823 INFO    MainThread:13351 [wandb_init.py:init():550] updated telemetry
2022-03-13 18:51:45,856 INFO    MainThread:13351 [wandb_init.py:init():581] communicating current version
2022-03-13 18:51:46,861 INFO    MainThread:13351 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-13 18:51:46,861 INFO    MainThread:13351 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-13 18:51:46,965 INFO    MainThread:13351 [wandb_init.py:init():624] starting run threads in backend
2022-03-13 18:51:48,738 INFO    MainThread:13351 [wandb_run.py:_console_start():1827] atexit reg
2022-03-13 18:51:48,739 INFO    MainThread:13351 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.REDIRECT
2022-03-13 18:51:48,742 INFO    MainThread:13351 [wandb_run.py:_redirect():1706] Redirecting console.
2022-03-13 18:51:48,744 INFO    MainThread:13351 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-13 18:51:48,744 INFO    MainThread:13351 [wandb_init.py:init():651] run started, returning control to user process
2022-03-13 18:54:40,342 INFO    MainThread:13351 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 1
2022-03-13 18:54:40,344 INFO    MainThread:13351 [wandb_run.py:_restore():1769] restore
2022-03-13 18:54:42,710 INFO    MainThread:13351 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 949
}

2022-03-13 18:54:42,934 INFO    MainThread:13351 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 949
}

2022-03-13 18:54:43,340 INFO    MainThread:13351 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 6933
}

2022-03-13 18:54:43,444 INFO    MainThread:13351 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 949
  total_bytes: 6961
}

2022-03-13 18:54:43,548 INFO    MainThread:13351 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6961
  total_bytes: 6961
}

2022-03-13 18:54:43,652 INFO    MainThread:13351 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6961
  total_bytes: 6961
}

2022-03-13 18:54:43,756 INFO    MainThread:13351 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6961
  total_bytes: 6961
}

2022-03-13 18:54:43,860 INFO    MainThread:13351 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6961
  total_bytes: 6961
}

2022-03-13 18:54:44,167 INFO    MainThread:13351 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6961
  total_bytes: 6961
}

2022-03-13 18:54:44,396 INFO    MainThread:13351 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6961
  total_bytes: 6961
}
local_info {
}

2022-03-13 18:54:45,905 INFO    MainThread:13351 [wandb_run.py:_append_files():2194] logging synced files
